docs/guide/config.qmd

```diff
@@ -134,11 +134,11 @@ license_check = cdp.CustomCheck(
 )
 ```
 
-For more details on what each parameter means, see the `CustomCheck`
-documentation. Specific to this example, the `type` is setting the
-identifier of the check to `only-mit` and the `jsonpath` is indicating
-to only check the `name` property of each license in the `licenses`
-property of the Data Package.
+For more details on what each parameter means, see the
+[`CustomCheck`](/docs/reference/CustomCheck.qmd) documentation. Specific
+to this example, the `type` is setting the identifier of the check to
+`only-mit` and the `jsonpath` is indicating to only check the `name`
+property of each license in the `licenses` property of the Data Package.
```

Comment:
I only added the link here. The diff looks big because of formatting the paragraph to fixed width.